functions/exercise6/scriptSpec.js

```diff
@@ -0,0 +1,28 @@
+/// <reference path="../../typings/globals/jasmine/index.d.ts" />
+/*globals findTheLongestWordWithinTheSentence*/
+
+describe("Finding the longest word within the sentence", function () {
+    it("string 'pig' gives  word 'pig'", function () {
+        var theLongestWord = findTheLongestWordWithinTheSentence("pig");
+        expect(theLongestWord).toEqual("pig");
+    });
+
+    it("string 'pig pig' gives  word 'pig pig'", function () {
```

Comment:
It should give 'pig', not 'pig pig' - test implementation is ok, this description is not correct.